Updated go script that doesn't try to update modules

gboone opened the pull request:
We don't need to update submodules any longer (thank goodness!) so we should have our go script attempt to.

Commit:
Updated go script that doesn't try to update modules

mbland commented on go:
```diff
@@ -90,20 +90,6 @@ def deploy(deploy_commands)
   puts 'Site built successfully.'
 end
 
-def deploy_staging
```
Don't forget to remove the commands from the `COMMANDS` hash below.

Commit:
Adds the hourglass page and fully removes submodule

- deinits the git submodule for data-private and removes the gitmodules
  file as well as the _data/private directory

gboone commented:
Alright, I think we're ready to merge this. :mouse:
